src/test/java/core/basesyntax/strategy/AddOperationHandlerTest.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax.strategy;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.dto.Transaction;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Operation;
+import org.junit.Before;
+import org.junit.Test;
+
+public class AddOperationHandlerTest {
+    private final OperationHandler addHandler = new AddOperationHandler();
+    private final Fruit apple = new Fruit();
+
+    @Test
+    public void test_AddingFruitsToStorage_OK() {
+        Storage.storage.put(apple, 10);
+        int expected = 10;
+        int actual = addHandler.apply(new Transaction(Operation.P, apple, 0));
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void test_AddingWrongData_OK() {
+        Storage.storage.put(apple, 10);
+        int expected = 0;
+        int actual = addHandler.apply(new Transaction(Operation.P, apple, 0));
+        assertNotEquals(expected, actual);
+    }
```

Comment:
this test does not look good
let's better check with `return` and `supply` operations separately